spec/services/export/report_spec.rb

```diff
@@ -318,6 +318,34 @@
     end
   end
 
+  describe "@activity_attributes" do
+    before do
+      allow(Export::ActivityAttributesColumns).to receive(:new).and_call_original
+    end
+
+    context "when the report is not ISPF" do
```

Comment:
Likewise here, I wonder if "not for ISPF activities" or "not associated with ISPF" might be clearer for future developers (i.e. ISPF is not itself an individual report)?

Reply:
Will reword this!